.github/actions/set-up/action.yml

```diff
@@ -0,0 +1,39 @@
+name: Free unused disk space
+description: Remove unused packages and files for Android build
+
+runs:
+  using: composite
+  steps:
+    - uses: nttld/setup-ndk@v1
+      id: setup-ndk
+      with:
+        ndk-version: r25c
+        link-to-sdk: true
+    - name: Setup Java
+      uses: actions/setup-java@v4
+      with:
+        distribution: 'temurin'
+        java-version: 17
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@v3
+    - run: |
+        sudo apt-get update --fix-missing
+        sudo apt-get remove -y '^dotnet-.*'
+        sudo apt-get remove -y '^llvm-.*'
+        sudo apt-get remove -y '^mysql-server-core-.*'
+        sudo apt-get remove -y '^postgresql-.*'
+        sudo apt-get remove -y azure-cli google-chrome-stable google-cloud-cli firefox powershell microsoft-edge-stable mono-devel
```

Comment:
https://www.ecloudcontrol.com/best-practices-to-reduce-docker-images-size/

Reply:
Updated!
Thank you :)